docs/content/getting-started/tuples-api-best-practices.mdx

```diff
@@ -29,14 +30,20 @@ You can use any string for user and object identifiers, however you should not i
 The documentation and samples uses first names and simple ids to illustrate easy-to-follow examples.
 :::
 
-## Always specify authorization model ID whenever possible
+## Always specify authorization model ID in requests
 
 It is strongly recommended that authorization model ID be specified in your Relationship Queries (such as [Check](./perform-check.mdx) and [ListObjects](../interacting/relationship-queries.mdx#listobjects)) and Relationship Commands (such as [Write](./update-tuples.mdx)).
 
 Specifying authorization model ID in API calls have the following advantages:
 1. Better performance as <ProductName format={ProductNameFormat.ShortForm}/> will not need to perform a database query to get the latest authorization model ID.
 2. Allows consistent behavior in your production system until you are ready to switch to the new model.
 
+## Specify the desired consistency in requests
+
+It is strongly recommended that you specify the `Consistency` value in each individual request, depending on whether you prefer speed (MINIMIZE_LATENCY) or correctness (MAXIMIZE_CONSISTENCY). The default is MINIMIZE_LATENCY.
```

Comment:
This is a good summary but again, is there a consistency section we can link to?